app/javascript/util/languageUtil.tsx

```diff
@@ -228,14 +235,13 @@ export function defaultIfNotTranslated(
  */
 export function localizedFormat(date: string, format: string): string {
   const lang = getCurrentLanguage(window.location.pathname)
-
-  return dayjs(date).locale(dayJsLocales[lang]).format(format)
+  return dayjs(date).tz().locale(dayJsLocales[lang]).format(format)
 }
 
 // Time zone is assumed to be Pacific
 export const formatTimeOfDay = (time: string) => {
-  const formattedTime = dayjs(time).format("h:mm")
-  const hour = Number(dayjs(time).format("H"))
+  const formattedTime = dayjs(time).tz().format("h:mm")
```

Comment:
Why not use `LT` as the format here?  That will output the `AM/PM` automatically.